lib/html_tag.js

```diff
@@ -24,6 +24,7 @@ function htmlTag(tag, attrs, text, escape = true) {
     if (attrs[i] === null || typeof attrs[i] === 'undefined') result += '';
     else {
       if (i.match(regexUrl)) result += ` ${escapeHTML(i)}="${encodeURL(attrs[i])}"`;
+      else if (attrs[i] === true) result += ` ${escapeHTML(i)}`;
```

Comment:
https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes

According to the description of `Boolean attributes`, The value of an attribute can be omitted as well if it is the same as attribute name.

Reply:
` ${escapeHTML(i)}` only outputs ` async` not `async="true"`.

Reply:
@curbengh 

I mean `async="async"` `disabled="disabled"` should output `async` `disabled` as well, since `Boolean attributes` not only accept Boolean value but also String value.

Reply:
Added